apps/prairielearn/src/tests/sprocs/users_select_or_insert.test.js

```diff
@@ -82,10 +93,11 @@ describe('sproc users_select_or_insert tests', () => {
   step('user 1 updates institution_id', async () => {
     const user = {
       ...baseUser,
+      name: 'J.R. User',
       institution_id: '100',
     };
 
-    const result = await usersSelectOrInsert(user, 'Shibboleth');
+    const result = await usersSelectOrInsert(user, 'Shibboleth', '100');
```

Comment:
`'Shibboleth'` is going away, and at any rate, an institution ID will never be passed in to `users_select_or_insert` when legacy Shib auth is used, so this test doesn't reflect a real-world scenario. I'd recommend changing this to `'SAML'` instead.

Reply:
I did a search/replace of Shibboleth to SAML, and did that for all of them except for the Default institution (where Shibboleth was used probably because it's authn_provider_id=1 as a defacto default). I think that's probably OK here?